api/handler/model.go

```diff
@@ -1383,3 +1383,36 @@ func (h *ModelHandler) ServerlessStop(ctx *gin.Context) {
 	}
 	httpbase.OK(ctx, nil)
 }
+
+// GetServerless godoc
+// @Security     ApiKey
+// @Summary      get model serverless
+// @Tags         Model
+// @Accept       json
+// @Produce      json
+// @Param        namespace path string true "namespace"
+// @Param        name path string true "name"
+// @Param        current_user query string true "current_user"
+// @Success      200  {object}  string "OK"
+// @Failure      400  {object}  types.APIBadRequest "Bad request"
+// @Failure      500  {object}  types.APIInternalServerError "Internal server error"
+// @Router       /models/{namespace}/{name}/serverless [get]
+func (h *ModelHandler) GetDeployServerless(ctx *gin.Context) {
```

Comment:
1. Need change @Security     ApiKey to Jwt token
2. for latest authentication, we don't need add current_user any more.

Reply:
Add new api which anyone can access by model detail

Reply:
Update code.